<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authorization</class>
 <widget class="QDialog" name="authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="UserText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LoginImage">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>130</y>
     <width>126</width>
     <height>126</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width:200px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/profile.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(192, 192, 192, 240);
text-align: center;
}
QPushButton:pressed{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(128, 128, 128, 240);
text-align: center;
}</string>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </widget>
  <widget class="QLabel" name="BlurImage">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>110</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(192, 192, 192, 150);
border-radius: 30px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswdText">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe Print&quot;;
color: rgba(0, 0, 0, 150);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswdInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 100);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoDB">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]STATUS</string>
   </property>
  </widget>
  <widget class="QLabel" name="BgImage">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>0</y>
     <width>500</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SingInBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(192, 192, 192, 240);
text-align: center;
}
QPushButton:pressed{
background-color: rgba(255, 255, 255, 0);
border-radius: 10px;
color: rgba(128, 128, 128, 240);
text-align: center;
}</string>
   </property>
   <property name="text">
    <string>Sing In</string>
   </property>
  </widget>
  <zorder>BgImage</zorder>
  <zorder>BlurImage</zorder>
  <zorder>CreateBtn</zorder>
  <zorder>LoginImage</zorder>
  <zorder>UserInput</zorder>
  <zorder>PasswdInput</zorder>
  <zorder>infoDB</zorder>
  <zorder>UserText</zorder>
  <zorder>PasswdText</zorder>
  <zorder>SingInBtn</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
